CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [Racecar::Config] Pass config to `rdkafka-ruby` via `producer` and `consumer`
 * [Racecar::Config] Replace `max_fetch_queue_size` with `min_message_queue_size`
 * [Racecar::Config] Add `synchronous_commits` to control blocking of `consumer.commit` (default `false`)
+* [Racecar::Config] Add `raise_on_partition_eof` to raise a `Rdkafka::RdkafkaError` when reaching the current end of the partition. Will only be re-raised after new messages arrive. Previous value was `true` (default `false`)
```

Comment:
"Previous value was `true`" is irrelevant for a PR on a PR. As Racecar had no concept of a end-of-partition previously, this statement is rather irritating. 

```suggestion
* [Racecar::Config] Add `raise_on_partition_eof` to raise a `Rdkafka::RdkafkaError` when reaching the current end of the partition. Will only be re-raised after new messages arrive. (default `false`)
```